protocol/daemons/server/bridge.go

```diff
@@ -40,12 +30,7 @@ func (s *Server) AddBridgeEvents(
 	*api.AddBridgeEventsResponse,
 	error,
 ) {
-	// If the daemon is unable to report a response, there is either an error in the registration of
-	// this daemon, or another one. In either case, the protocol should panic.
-	// TODO(CORE-582): Re-enable this check once the bridge daemon is fixed in local / CI environments.
-	//if err := s.reportResponse(types.BridgeDaemonServiceName); err != nil {
-	//	panic(err)
-	//}
+	s.reportResponse(types.BridgeDaemonServiceName)
```

Comment:
Given that we don't know if the below logic will succeed or not at this point, should we move this `reportResponse` to right before the return statement?

Reply:
As of when we last touched this, the logic in reportResponse is only recording if the daemon successfully queries the server. We could definitely make a change here, but this is outside the scope of this work and, to be consistent, would probably require auditing other daemon gRPC query response code as well. I think it's better for now to leave the existing feature as is and stay focused on the health check feature.

Reply:
ICIC - if it's only a few lines, I can just move it to the end of each method. Does that fully capture the concern? I'm sorry about that.